docs/next-release-notes.md

```diff
@@ -0,0 +1,496 @@
+
+### Idle configs for the Supervisor
+
+You can now configure the following properties:
+
+| Property | Description | Default |
+| - | - | -|
+|`druid.supervisor.idleConfig.enabled`| (Cluster wide) If `true`, supervisor can become idle if there is no data on input stream/topic for some time.|false|
+|`druid.supervisor.idleConfig.inactiveAfterMillis`| (Cluster wide) Supervisor is marked as idle if all existing data has been read from input topic and no new data has been published for `inactiveAfterMillis` milliseconds.|`600_000`|
+| `inactiveAfterMillis` | (Individual Supervisor) Supervisor is marked as idle if all existing data has been read from input topic and no new data has been published for `inactiveAfterMillis` milliseconds. | no (default == `600_000`) |
+
+https://github.com/apache/druid/pull/13311
+
+### cachingCost balancer strategy
+
+The `cachingCost` balancer strategy now behaves more similarly to cost strategy. When computing the cost of moving a segment to a server, the following calculations are performed:
+
+- Subtract the self cost of a segment if it is being served by the target server
+- Subtract the cost of segments that are marked to be dropped
+
+https://github.com/apache/druid/pull/13321
+
+### New metrics for streaming ingestion
+
+The following metrics related to streaming ingestion have been added:
+
+- `ingest/kafka/partitionLag`: Partition-wise lag between the offsets consumed by the Kafka indexing tasks and latest offsets in Kafka brokers. 
+- `ingest/kinesis/partitionLag/time`: Partition-wise lag time in milliseconds between the current message sequence number consumed by the Kinesis indexing tasks and latest sequence number in Kinesis.
+- `ingest/pause/time`: Milliseconds spent by a task in a paused state without ingesting.|dataSource, taskId| < 10 seconds.|
+
+https://github.com/apache/druid/pull/13331
+https://github.com/apache/druid/pull/13313
+
+### Backoff for HttpPostEmitter
+
+The `HttpPostEmitter` option now has a backoff. This means that there should be less noise in the logs and lower CPU usage if you use this option for logging. 
+
+https://github.com/apache/druid/pull/12102
+
+### taskActionType dimension for task/action/run/time metric
+
+The `task/action/run/time` metric for the Indexing service now includes the `taskActionType` dimension.
+
+https://github.com/apache/druid/pull/13333
+
+### DumpSegment tool for nested columns
+
+The DumpSegment tool can now be used on nested columns with the `--dump nested` option. 
+
+For more information, see [dump-segment tool](https://druid.apache.org/docs/latest/operations/dump-segment).
+
+https://github.com/apache/druid/pull/13356
+
+### Segment assignment
+
+You can now use a round-robin segment strategy to speed up initial segment assignments.
+
+Set `useRoundRobinSegmentAssigment` to `true` in the Coordinator dynamic config to enable this feature.
+
+https://github.com/apache/druid/pull/13367
+
+### Segment balancing
```

Comment:
Do you think it would be better to club the `segment balancing`, `segment replication` and `segment assignment` under one heading? They are all related coordinator-side changes which improve segment loading and balancing.

Reply:
I'll move them under an H3 called `Segments` and put assignment, balancing, replication, and discovery as H4s underneeth